main_test.go

```diff
@@ -510,12 +536,90 @@ var testCases = []TestCase{
 			"leslie": changePasswordPath,
 		},
 	},
+	{
```

Comment:
I don't think these test cases really verify anything. What was your goal in creating them?